common.js

```diff
@@ -99,9 +100,21 @@ function generateFinalPath (opts) {
   return path.join(opts.out || process.cwd(), generateFinalBasename(opts))
 }
 
-function subOptionWarning (properties, optionName, parameter, value) {
+function info (message, quiet) {
+  if (!quiet) {
+    console.error(message)
```

Comment:
I didn't. Explanation is here: https://github.com/electron-userland/electron-packager/pull/228#issuecomment-168066128